react-native/redux/custom-redux-form-field.md

```diff
@@ -0,0 +1,104 @@
+# [MO] How to use redux-form with custom fields (~10min)
+
+## Owner : [Darius Afchar](https://github.com/DariusAf)
+
+*Why would you want to do that ?*
+
+Redux-form is great and can automate a lot of things as form-validation, focus events, storage synchronisation... But it can be a bit tricky if you want to add a custom component that includes many fields whose values have to be aggregated, or an external component, for instance a date-picker.
```

Comment:
- What is redux form first.
- What are the strength ? => bullet points
- Why this article 

Basically, is what you said but I guess you can find a less convoluted way to say it.